applications/CompressiblePotentialFlowApplication/python_scripts/stochastic_potential_flow_response.py

```diff
@@ -3,13 +3,14 @@
 import KratosMultiphysics.CompressiblePotentialFlowApplication as KCPFApp
 from KratosMultiphysics.response_functions.response_function_interface import ResponseFunctionInterface
 import KratosMultiphysics.CompressiblePotentialFlowApplication.potential_flow_analysis as potential_flow_analysis
+import KratosMultiphysics.MappingApplication
 
 # Import Kratos, XMC, PyCOMPSs API
 import KratosMultiphysics.MultilevelMonteCarloApplication
 import xmc
 import xmc.methodDefs_momentEstimator.computeCentralMoments as mdccm
 from exaqute import get_value_from_remote
-import json, os
+import json, os, math, time
```

Comment:
I suggest to first import Python libraries and then import Kratos/xmc libraries. @roigcarlo taught me so time ago.